keithleyClient/keithleyInterface.py

```diff
@@ -68,7 +68,7 @@ def read(self,minLength=0):
             time.sleep(self.readSleepTime)
             i+=1
         ts =time.time()
-        maxTime = 20
+        maxTime = 300
```

Comment:
This was required because our power supplies were having trouble with the auto-range.  With a shorter timeout, the IV scans were aborted prematurely while the power supplies attempted to find the correct range with which to measure the current.